CONTRIBUTING.rst

```diff
@@ -225,6 +240,8 @@ The QA process (``make qa``) consists of multiple parts:
 
 - linting (``make lint``)
 
+- checking format (``make format``)
```

Comment:
Was renamed.

```suggestion
- checking format (``make check-format``)
```

And we should perhaps mention that this step won't yet succeed…